ext/datasource/helper.go

```diff
@@ -209,7 +209,7 @@ func FrequencyParamsRulesJsonConverter(src []byte) (interface{}, error) {
 		freqRule := &hotspot.Rule{
```

Comment:
Keep naming conventions
Use hotspotRule instead of freq param rule/Freq Rule/....